### ISSUE TITLE
Gradle 7.0 throws several build validation errors

### ISSUE THREAD

**Issue by sschuberth** [palantir/gradle-graal#421]
## What happened?

I tried using plugin version 0.7.2 in [our project](https://github.com/oss-review-toolkit/ort/tree/cli-graalvm) which uses Gradle 7.0.

## What did you want to happen?

Gradle's new [build validation](https://docs.gradle.org/7.0/release-notes.html#build-reliability) throws several errors at me:

```
* What went wrong:
Some problems were found with the configuration of task ':cli:nativeImage' (type 'NativeImageTask').
  - Type 'NativeImageTask' property 'architectureSpecifiedOutputExtension' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'NativeImageTask' property 'cacheDir' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'NativeImageTask' property 'executable' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'NativeImageTask' property 'jarFile' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'NativeImageTask' property 'options' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
```

**Comment by nhoughto** [comment]
Would be nice if this was fixed, blocking our upgrade to gradle 7. PRs welcome?

**Comment by nhoughto** [comment]
This was fixed 2 months ago appr but not released until 0.8.0, in commit 
https://github.com/palantir/gradle-graal/commit/2860547770e11a71bf8904344e1c25b601d3a28f

**Comment by sschuberth** [comment]
I can confirm these errors to be gone in release 0.9.0.
